lib/cmds/data_lake_cmds/get-schema.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+
+const { get } = require('../../api');
+const print = require('../../print');
+
+exports.command = 'get-schema <projectId>';
+exports.desc = 'Fetch a single query execution.';
+exports.builder = yargs => {
+  yargs.positional('projectId', {
+    describe: 'The ID of the project to fetch the schema from.',
+    type: 'string'
+  }).option('table', {
+    alias: 't',
+    type: 'string',
+    describe: 'Name of the table to fetch the schema of.',
```

Comment:
Are the table names dynamic?  If not, you can add an enumerated set to aid a user.

Reply:
The tables are dynamic in-that a project will only have tables for which there is data, e.g. a project that only has patient data will not have the `variant` table.

Reply:
Yeah, so would something like this help:

``` 
choices: ['variant', 'patient', 'gene', ...],
describe: 'Name of the table to fetch the schema of (The possible list of choices are provided. Note that not every project will have every table)',
```

Reply:
Done